RF_Receiver/RF_Receiver.ino

```diff
@@ -171,6 +171,8 @@ void setup() {
 	cc1101::setup();
 	#endif
   	initEEPROM();
+	musterDec.MsMoveCount = 0;
```

Comment:
Der Grund dafür ist, daß ich es nicht weiß wie man es in der Klasse sauber initialisiert.
Ich habe es so versucht:
> uint8_t MsMoveCount = 0;

Da bekomme ich aber in der Arduino IDE ein warning:
```
signalDecoder.h:85:24: warning: non-static data member initializers only available with -std=c++11 or -std=gnu++11 [enabled by default]
  uint8_t MsMoveCount = 0;
```